app/Models/Signup.php

```diff
@@ -45,6 +46,20 @@ class Signup extends Model
         'campaign_id', 'updated_at', 'northstar_id', 'id', 'quantity', 'source',
     ];
 
+    /**
+     * Attributes that we can sort by with the '?orderBy' query parameter. These
+     * should not contain any sensitive or private data (as they'll be encoded
+     * in cursors).
+     *
+     * This array is manually maintained. It does not necessarily mean that
+     * any of these are actual indexes on the database... but they should be!
+     *
+     * @var array
+     */
+    public static $sortable = [
+        'campaign_id', 'updated_at', 'northstar_id', 'id', 'quantity', 'source',
```

Comment:
What does the 'id' track? Is it the page number?

Reply:
This is the signup's ID, which is used to uniquely identify it (such as referring to which signup a given post was created under).